src/StreamService/Properties/launchSettings.json

```diff
@@ -0,0 +1,22 @@
+﻿{

```

Comment:
are you sure this file should look like this?

Reply:
fixed